diff --git a/colors/one.vim b/colors/one.vim
index 54747c3..69e594b 100644
--- a/colors/one.vim
+++ b/colors/one.vim
@@ -308,7 +308,7 @@ if has('gui_running') || has('termguicolors') || &t_Co == 88 || &t_Co == 256
     let s:hue_6   = ['#d19a66', '173'] " orange 1
     let s:hue_6_2 = ['#e5c07b', '180'] " orange 2
 
-    let s:syntax_bg     = ['#282c34', '16']
+    let s:syntax_bg     = ['#000000', '16']
     let s:syntax_gutter = ['#636d83', '60']
     let s:syntax_cursor = ['#2c323c', '16']
 
@@ -444,15 +444,15 @@ if has('gui_running') || has('termguicolors') || &t_Co == 88 || &t_Co == 256
   " }}}
 
   " Diff highlighting -------------------------------------------------------{{{
-  call <sid>X('DiffAdd',     s:hue_4, s:visual_grey, '')
-  call <sid>X('DiffChange',  s:hue_6, s:visual_grey, '')
-  call <sid>X('DiffDelete',  s:hue_5, s:visual_grey, '')
-  call <sid>X('DiffText',    s:hue_2, s:visual_grey, '')
-  call <sid>X('DiffAdded',   s:hue_4, s:visual_grey, '')
-  call <sid>X('DiffFile',    s:hue_5, s:visual_grey, '')
-  call <sid>X('DiffNewFile', s:hue_4, s:visual_grey, '')
-  call <sid>X('DiffLine',    s:hue_2, s:visual_grey, '')
-  call <sid>X('DiffRemoved', s:hue_5, s:visual_grey, '')
+  call <sid>X('DiffAdd',     s:hue_4, s:syntax_bg, '')
+  call <sid>X('DiffChange',  s:hue_6, s:syntax_bg, '')
+  call <sid>X('DiffDelete',  s:hue_5, s:syntax_bg, '')
+  call <sid>X('DiffText',    s:hue_2, s:syntax_bg, '')
+  call <sid>X('DiffAdded',   s:hue_4, s:syntax_bg, '')
+  call <sid>X('DiffFile',    s:hue_5, s:syntax_bg, '')
+  call <sid>X('DiffNewFile', s:hue_4, s:syntax_bg, '')
+  call <sid>X('DiffLine',    s:hue_2, s:syntax_bg, '')
+  call <sid>X('DiffRemoved', s:hue_5, s:syntax_bg, '')
   " }}}
 
   " Asciidoc highlighting ---------------------------------------------------{{{
